dyninstAPI/h/BPatch_snippet.h

```diff
@@ -238,8 +238,10 @@ class BPATCH_DLL_EXPORT BPatch_constExpr : public BPatch_snippet {
     //  Creates a representation of a (signed long) value
     BPatch_constExpr(signed long value);
     
-    //  Creates a representation of an (unsigned long) value
```

Comment:
Surely you meant `sizeof(void*)`, but I think markdown ate your unquoted `*`.  And yes, `uintptr_t` is the portable way to match an integer to pointer size, standardized in C++11.
(Although [here](http://en.cppreference.com/w/cpp/types/integer) it's marked optional? Standards are weird...)

But changing the interface here seems wrong.  If I really wanted `unsigned long` with no relation to pointers, what am I supposed to use?  There's an actual `void*` variant for pointers below, and `long long` too.  Perhaps this just needs a new `unsigned long long` variant to satisfy LLP64 needs.